articles/logs/references/log-data-retention.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Log Data Retention
+description: Describes how long log data is stored depending on your Auth0 plan.
+topics:
+  - logs
+  - log-data
+contentType:
+  - reference
+useCase:
+  - manage-logs
+---
+# Log Data Retention
```

Comment:
```
# Log Data Retention

The retention period for your Auth0 logs is determined by your subscription level:

Plan | Log Retention
-----|--------------
Free | 2 days
Developer | 2 days
Developer Pro | 10 days
Enterprise | 30 days

For longer-term storage or analytics, you can export log data using a log export extension, rules, or the [AWS EventBridge Integration](/integrations/aws-eventbridge).
```

Reply:
Kinda shortening up the page, could add the example uses to the main index?

Reply:
I'm going to keep them separate and add more crap from the Arch Scenarios here.